app/src/main/res/layout/activity_hospital.xml

```diff
@@ -218,6 +218,13 @@
 
     </LinearLayout>
 
+    <TextView
+        android:id="@+id/amountofoverduedevices"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textAlignment="center"
+        android:textSize="15dp"/>
```

Comment:
Text sizes should be measured in "sp", not "dp"

Reply:
done.